app/components/form.tsx

```diff
@@ -56,7 +55,7 @@ export const Label = ({
             textAlign: "left",
             fontFamily: "body",
             pr: 1,
-            fontSize: [3],
+            fontSize: smaller ? [2, 2, 2] : [3, 3, 3],
```

Comment:
Won't `smaller ? 2 : 3` work in the same way?

Reply:
Yes, thanks, don't know why I left this... FIxed ;)